Assets/Scripts/Models/World.cs

```diff
@@ -166,7 +166,7 @@ void SetupWorld(int width, int height)
                 tiles[x, y].Room = GetOutsideRoom(); // Rooms 0 is always going to be outside, and that is our default room
             }
         }
-
+        new NeedActions ();
```

Comment:
This doesnt look right. Why are you instantiating an object and then never use it=